db/generateSeedScript.js

```diff
@@ -0,0 +1,111 @@
+
+// product types
+var types = ["Men's Shoe", "Women's Shoe"];
+
+// a bunch of random style descriptors
+var styles = ["Team Red/Black/Bright Crimson","Platinum Tint/Black/Pure Platinum/Kumqu","White/Metallic Silver/Bright Crimson/Metallic Silver","Black/Game Royal","Black/Black/White/Black","Platinum Tint/Black/University Red","Pacific Blue/Mystic Red/Black/Pale Ivory","Geode Teal/Light British Tan/Kumquat/Pale Ivory","Black/Metallic Silver/Wolf Grey/Black","White/Platinum Tint/White","Black/Court Purple/Black/Jade Horizon","Light Bone/Metallic Gold/University Red/Summit White","Green Strike/Black/Watermelon","White/Metallic Silver/Light Aqua/Black","Platinum Tint/Black/University Red","Summit White/Solar","Red/White/Light Blue Fury","Blue/Photo Blue/White/Black","Royal Pulse/Light Aqua/Ember Glow/Oil Grey","BleachedCoral/White-AphidGreen","White/Anthracite/Black","White/Metallic Silver/Max Orange/White","Black/Metallic Hematite/Max Orange/Black","Black/Amarillo/University Red/Black","Black/Black","White/Neon Yellow/Dark Grey/Atom Red","Black/Dark Grey/White/Black","Black/Racer Blue/Wolf Grey/Hyper Jade","Orange Peel/Pure Platinum/White/Aphid Green","Habanero Red/White/Metallic Silver/Black","Midnight Navy/Wolf Grey/White/Black","Bordeaux/Sail/Hyper Jade/Bordeaux","Black/Mineral Teal/Black/Photo Blue","Pacific Blue/Deep Royal Blue/White/White","White/Metallic Silver/University Red","Team Gold/Black/White/University Red","Habanero Red/Oil Grey","Black/Bordeaux/University Gold/Anthracite","Thunder Grey/Black/Wolf Grey/Hot Punch","Racer Blue/Obsidian/White","Plum Eclipse/Pumice/Burgundy Ash/University Blue"]
+
+// style adjustment pricing 
+var priceAdjust = [-20, 20, -10, -10, -10, -10, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 10, 10, 10, 10, 10, 20, 20]
+
+var sizes = {};
+sizes["Men's Shoe"] = ['6','6.5','7','7.5','8','8.5','9','9.5','10','10.5','11','11.5','12','12.5','13','14','15'];
+sizes["Women's Shoe"]  = ['5','5.5','6','6.5','7','7.5','8','8.5','9','9.5','10','10.5','11','11.5','12'];
+
+var chars = ['A','B','C','D','E','F','G','H','I','J','K','L','M','N','O','P','Q','R','S','T','U','V','W','X','Y','Z']
+
+var inventory = [0,0,0,0,0,0,0,0,0,0,1,1,2,2,3,3,4,4,5,5,6,6,7,7,8,8,9,9,10,10]
+
+// a more flexible randomizer that takes arrays and numbers
+var rnd = n => Array.isArray(n) ? n[ rnd(n.length) - 1] : Math.floor( Math.random() * n ) + 1
+
+// generate a random style code ( "AB-1234-styleId" )
+var getStyleCode = id => rnd(chars) + rnd(chars) + (rnd(9000) + 1000) + "-" + id;
+
+// from the base price of a product, returns an array containing plausible base & sale prices for a style 
+var getStylePrice = base => {
+  var adjust = base + rnd( priceAdjust ); // set the adjusted price for this style
+  return rnd(2) == 2 
```

Comment:
i like this way of getting a 50% chance of either or

Reply:
Sweet! Yeah - once I made that randomizer function, I kept coming up with lots of uses for it. I noticed that whenever I used the native JS version it just felt so clunky that I didn't want to.